include/wlr/render/wlr_renderer.h

```diff
@@ -127,4 +127,10 @@ int wlr_renderer_get_drm_fd(struct wlr_renderer *r);
  */
 void wlr_renderer_destroy(struct wlr_renderer *renderer);
 
+/**
+ * Return the buffer of called 'wlr_renderer_bind_buffer' after.
+ * If no a active buffer then return NULL.
+ */
+struct wlr_buffer *wlr_renderer_get_current_buffer(struct wlr_renderer *renderer);
```

Comment:
This is only useful if the caller doesn't know about the `wlr_buffer`. Instead of adding this function, I'd prefer to make it easier for compositors to allocate their own buffers and call `wlr_renderer_begin_with_buffer`.

Reply:
Yes, wlr_renderer_begin_with_buffer is better, do you have a plan about it? I look the https://github.com/swaywm/wlroots/pull/2903, when remove the wlr_output_attach_render, What kind of solution will replace it?

Reply:
> What kind of solution will replace it?

Instead, compositors will be able to create their own swapchain with `wlr_swapchain_create`, acquire a buffer from it, render to it via `wlr_renderer_begin_with_buffer`, and attach it to the output via `wlr_output_attach_buffer`. We may introduce additional helpers to make this whole dance easier.

Note, `wlr_output_attach_render` will likely stay around for a while, removing it would be a very hard breaking change for all compositors. But the old and new APIs can coexist.

Reply:
OK, I have remove the interface, wait the wlr_renderer_begin_with_buffer instead it.